pkgs/applications/networking/syncthing/default.nix

```diff
@@ -3,14 +3,14 @@
 let
   common = { stname, target, postInstall ? "" }:
     buildGoPackage rec {
```

Comment:
If you use buildGoModule you can skip adding the deps.nix

Reply:
When I removed `goDeps = ./deps.nix` it failed to build.

Reply:
with these changes it build for me: https://gist.github.com/Lassulus/fc2cc45fc5caffa4e76885644c26e0fb

Reply:
Ah, I see. Do you have a patch for this? I'll apply it and try it out. The patch above is missing the header, so Git won't apply it.

Reply:
ah sure: http://paste.krebsco.de/9mntuseY/+inline